sentry/src/test/java/io/sentry/SentryClientTest.kt

```diff
@@ -1038,39 +1039,55 @@ class SentryClientTest {
         fixture.getSut().captureTransaction(transaction, createScopeWithAttachments(), null)
 
         verifyAttachmentsInEnvelope(transaction.eventId)
-        assertFails { verifyProfilingTraceInEnvelope(transaction.eventId) }
+        assertFails { verifyProfilingTraceInEnvelope(SentryId(fixture.profilingTraceData.profileId)) }
     }
 
     @Test
     fun `when captureTransaction with attachments and profiling data`() {
         val transaction = SentryTransaction(fixture.sentryTracer)
-        fixture.getSut().captureTransaction(transaction, null, createScopeWithAttachments(), null, fixture.profilingTraceData)
-        verifyAttachmentsInEnvelope(transaction.eventId)
-        verifyProfilingTraceInEnvelope(transaction.eventId)
+        val client = fixture.getSut()
+        client.captureTransaction(transaction, null, createScopeWithAttachments(), null)
+        client.captureProfile(fixture.profilingTraceData)
+        verifyProfilingTraceInEnvelope(SentryId(fixture.profilingTraceData.profileId))
```

Comment:
`m`: I think we should split that test case up into two test cases. Previously, we had one call on the client. Now we have two and only verify for the profiling trace in the envelope. 

Something like
```Kotlin
client.captureTransaction(transaction, null, createScopeWithAttachments(), null)
verifyAttachmentsInEnvelope(transaction.eventId)
```
and
```Kotlin
client.captureProfile(fixture.profilingTraceData)
verifyProfilingTraceInEnvelope(SentryId(fixture.profilingTraceData.profileId))
```

That also applies to some of the other test cases.

Reply:
yep. Actually, there were already those tests, so i just removed the transactions from the captureProfile tests